atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -68,3 +71,202 @@ export const fatalityGridTableColumns = {
+            },
+          ],
+        },
+      },
+      {
+        id: "cyclist",
+        label: "Cyclist Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { person: { unit: { unit_desc_id: { _eq: 3 } } } }, { primaryperson: { unit: { unit_desc_id: { _eq: 3 } } } } ]": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "pedestrian",
+        label: "Pedestrian Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { person: { unit: { unit_desc_id: { _eq: 4 } } } }, { primaryperson: { unit: { unit_desc_id: { _eq: 4 } } } } ]": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "scooter",
+        label: "Scooter-rider Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { _and: [ { person: { unit: { unit_desc_id: { _eq: 177 } } } }, { person: { unit: { veh_body_styl_id: { _eq: 177 } } } } ] }, { _and: [ { primaryperson: { unit: { unit_desc_id: { _eq: 177 } } } }, { primaryperson: { unit: { veh_body_styl_id: { _eq: 177 } } } } ] } ]": null,
+            },
+          ],
+        },
+      },
+    ],
+  },
+  groupStatus: {
+    icon: "clipboard",
+    label: "Status",
+    filters: [
+      {
+        id: "status_open_short_term",
+        label: "Open - Short Term",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 1 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_open_long_term",
+        label: "Open - Long Term",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 2 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_rec",
+        label: "Closed - No Action Recommended",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 3 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_rec_implemented",
+        label: "Closed - Recommendation Implemented",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 4 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_closed_no_action_taken",
+        label: "Closed - No Action Taken",
+        filter: {
+          where: [
+            {
+              "recommendation: { recommendation_status_id: { _eq: 5 } } ": null,
+            },
+          ],
+        },
+      },
+      {
+        id: "status_null",
+        label: "Null",
```

Comment:
Just going to throw out another suggestion of "Not provided" because both null and undefined are both data types in JS. Not a strong feeling here because I think any word that Xavier and the VZ team can easily interpret as "no value provided by CRIS" is the most important part.

It gets confusing for me in the UI sometimes because CRIS lookup tables provide lots of values like "NO DATA", "NONE", "NOT REPORTED", etc that I confuse with words that we set in the code.

Reply:
Thanks for the input Mike! Also for context, the recommendation status is a value that the VZ team gives to crashes with fatalities (part of the Fatality Review Board feature) and its not something thats provided by CRIS. So if a fatality is missing a recommendation status its because the VZ team hasnt given it one yet. Im still not sure what would be the best toggle label to describe that, I personally like "None" but open to more opinions!

Reply:
Perhaps, "Missing Values" or "Not Available."